lib/Environment.php

```diff
@@ -386,6 +386,9 @@ public function setCurrentFileName(string $filename): void
         $this->currentFileName = $filename;
     }
 
+    /**
+     * Returns the currently-parsed filename, without the extension (e.g. "index" or "subdir/file")
```

Comment:
So this file name is relative to something in particular? The working directory, or maybe the directory of some document being parsed?